src/test/java/org/candlepin/dto/api/v1/ConsumerTranslatorTest.java

```diff
@@ -226,8 +232,18 @@ protected void verifyOutput(Consumer source, ConsumerDTO dest, boolean childrenG
                 ConsumerType ctype = this.mockConsumerTypeCurator.getConsumerType(source);
                 this.consumerTypeTranslatorTest.verifyOutput(ctype, dest.getType(), true);
 
-                Environment environment = this.mockEnvironmentCurator.getConsumerEnvironment(source);
-                this.environmentTranslatorTest.verifyOutput(environment, dest.getEnvironment(), true);
+                if (source.getEnvironmentIds() != null) {
+                    for (Map.Entry<String, String> sourceEnvId : source.getEnvironmentIds().entrySet()) {
```

Comment:
This test should be verifying two extra things: first that the destination consumer has translated exactly as many environments as the source consumer had (see `ProductTranslatorTest.java` verifying Branding as an example),  and second that it should also check that the order in which they were translated was preserved (this is a unique use case for this feature which we should be testing)

Reply:
Done.